### PR TITLE
Always ref-sink after g_object_new() if the type inherits from GIniti…

### PR DESCRIPTION
**Pull request by sdroege**
…allyUnowned

Independent of it still being actually floating or not.

Fixes https://github.com/gtk-rs/glib/issues/483

### CODE REVIEW

File: src/object.rs [hunk 1]
```diff
@@ -882,7 +882,7 @@ impl Object {
             let ptr = gobject_sys::g_object_newv(type_.to_glib(), params_c.len() as u32, mut_override(params_c.as_ptr()));
             if ptr.is_null() {
                 Err(glib_bool_error!("Can't instantiate object"))
-            } else if gobject_sys::g_object_is_floating(ptr) != glib_sys::GFALSE {
+            } else if gobject_sys::g_type_is_a(type_.to_glib(), gobject_sys::g_initially_unowned_get_type()) != glib_sys::GFALSE {
```
Comment (antoyo) [inline src/object.rs]: Could we do something like:
```rust
type::is_a::<glib::object::InitiallyUnowned>()
```
?

Comment (sdroege) [reply src/object.rs]: Yes, does that work?

Comment (antoyo) [reply src/object.rs]: I don't know. I can try it if you want.

Comment (antoyo) [reply src/object.rs]: The following works:
```rust
type_.is_a(&glib::object::InitiallyUnowned::static_type())
```

Comment (sdroege) [reply src/object.rs]: Thanks, updated :)

**Comment by sdroege** [comment]
Will merge once the CI is through

**Pull request merged**